### PR TITLE
Update growth team strategy

### PR DESCRIPTION
**Pull request by a-bergevin**
adding our team strategy to the handbook

### CODE REVIEW

File: content/strategy-goals/strategy/cloud/growth-team/index.md [hunk 1]
```diff
@@ -1,47 +1,68 @@
...
+- Improve _retention & engagement_ through early onboarding to the product. Both new website visitors and new customers will need personalized guided product experiences that help them reach the aha moment where they understand the value of the product and have it integrated into their environment and workflows. We will partner with existing customers and enterprise users to discover the most optimal way to onboard to SG.[^1]
+- Improve _cohesion_ between product and marketing/sales language by aligning messaging on sourcegraph.com with the five use cases. Customer feedback and traction in sales have shown the use cases resonate with our customers. Though the language around the use cases is apparent in the newly refreshed marketing site, these value stories can be better told throughout the product experience - particularly for new users.
 
-### Themes
+### Goals
 
-TODO
+- Improve visitor registration rate (# of visitors who sign up / # of users visiting over the period)
+- Reduce 1 and done rate
```
Comment (erzhtor) [inline content/strategy-goals/strategy/cloud/growth-team/index.md:48]: I'm not sure if I fully understand this. Can you, please, elaborate on what this means (maybe some examples)?

P.S: I guess it is about doing some action once and never again. But still not sure what kind of actions/things we mean in these cases.

**Review by erzhtor — APPROVED** [review]
Awesome! 🔥 Thanks for tackling and updating the handbook! 👍

File: content/strategy-goals/strategy/cloud/growth-team/index.md (hunk 1 above)
Comment (a-bergevin) [reply content/strategy-goals/strategy/cloud/growth-team/index.md:48]: Sure @erzhtor it means visits the website once and never returns. So we may calculate it as waiting 30 days and not seeing them come back, for example. It's related tor retention, but measured slightly differently. Does that make sense?

**Comment by erzhtor** [comment]
@a-bergevin, can we continue merging this PR, or is there anything to update/change?

**Comment by a-bergevin** [comment]
@erzhtor yes please merge! Thanks!

**Pull request merged**